tools/scripts/tidy.toml

```diff
@@ -24,8 +24,12 @@ script_runner = "@duckscript"
 script = '''
 exit_on_error true
 
+# Check files matching these patterns.
 glob_pattern_array = array "./**/*.rs" "./**/*.yml" "./**/*.toml" "./**/*.rst" "./**/*.bat"
 
+# Skip the files matching these patterns.
+glob_skip_pattern_array = array "target/**/*.rs" "ffi/diplomat/cpp/docs/source/*.rst" "ffi/diplomat/wasm/docs/*.rst"
```

Comment:
it should just be everything in the docs folders

Reply:
I include only `*.rst` because we should check `ffi/diplomat/cpp/docs/source/conf.py` and `ffi/diplomat/cpp/docs/make.bat` for example. Do you recommend we use `"ffi/diplomat/cpp/docs/*" "ffi/diplomat/wasm/docs/*"`?

Reply:
I don't think it's important to check that code, it's generated boilerplate.

Reply:
OK. I've add a commit to skip all files under `docs`. Also, `char16trie` crate needs to be fixed.